pkgs/development/python-modules/python-logstash/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi
+, pika }:
+
+buildPythonPackage rec {
+  pname = "python-logstash";
+  version = "0.4.6";
+
+  disabled = isPyPy;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "13763yx0k655y0c8gxv7jj6cqp45zypx2fmnc56jnn9zz1fkx50h";
+  };
+
+  nativeBuildInputs = [ pika ];
+
+  meta = with lib; {
+    description = "Python logging handler for Logstash. http://logstash.net/";
```

Comment:
Putting links is the description isn't helpful.
That would be better to have in `homepage`.

Also having `maintainers` set is good thing to have too.

Reply:
Maybe also add a `repositories.git` with `https://github.com/vklochan/python-logstash`.